packages/core/handlers/backend-utils.js

```diff
@@ -1,24 +1,40 @@
-const { createFriggBackend, Worker } = require('@friggframework/core');
 const { findNearestBackendPackageJson } = require('@friggframework/core/utils');
 const path = require('node:path');
 const fs = require('fs-extra');
+const { Router } = require('express');
 
-const backendPath = findNearestBackendPackageJson();
-if (!backendPath) {
-    throw new Error('Could not find backend package.json');
-}
 
-const backendDir = path.dirname(backendPath);
-const backendFilePath = path.join(backendDir, 'index.js');
-if (!fs.existsSync(backendFilePath)) {
-    throw new Error('Could not find index.js');
-}
+/**
+ * Loads the App definition from the nearest backend package
+ * @function loadAppDefinition
+ * @description Searches for the nearest backend package.json, loads the corresponding index.js file,
```

Comment:
As we restructure by removing the "frontend" directory from the default frigg structure, this makes me realize we need to rethink defaults and likely provide an override somewhere to point at the frigg entry point/definition. Frigg config comes to mind, lots of JS libraries and tools do that ("use the Frigg config file to tell us where to look/find things").